PennMobile/Login/PathAtPennNetworkManager.swift

```diff
@@ -149,6 +198,20 @@ extension PathAtPennNetworkManager {
                 $0.split(separator: "|").first
             }
 
+            // Side effect: update Penn Course Plan Path Registration
+            try await withCheckedThrowingContinuation { continuation in
+                CourseAlertNetworkManager.instance.updatePathRegistration(srcdb: srcdb, crns: crns) { success, error in
+                    if let error = error {
+                        continuation.resume(throwing: error)
+                    } else if success {
+                        continuation.resume(returning: ())
+                    } else {
+                        // Handle the case where success is false and there's no error
+                        continuation.resume(throwing: NSError(domain: "UpdatePathRegistrationError", code: 0, userInfo: [NSLocalizedDescriptionKey: "Unknown error occurred"]))
+                    }
+                }
+            }
```

Comment:
Should we wrap this in a `Task`? This would make the function not wait for the registration to be updated before returning.